Formula/s/siril.rb

```diff
@@ -6,6 +6,13 @@ class Siril < Formula
   license "GPL-3.0-or-later"
   head "https://gitlab.com/free-astro/siril.git", branch: "master"
 
+  livecheck do
+    url "https://gitlab.com/api/v4/projects/free-astro%2Fsiril/releases"
+    strategy :json do |json|
+      json.map { |release| release["tag_name"] }
+    end
```

Comment:
```suggestion
    url "https://siril.org/download/"
    regex(/href=.*?siril[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Checking the GitLab repository doesn't align with the `stable` source. In this case, we should check the first-party download page, which links to the `stable` tarball.